admin/server/server.go

```diff
@@ -235,6 +235,24 @@ func (s *Server) HTTPHandler(ctx context.Context) (http.Handler, error) {
 	// Add Github-related endpoints (not gRPC handlers, just regular endpoints on /github/*)
 	s.registerGithubEndpoints(mux)
 
+	// Add biller webhook handler if any
+	if s.admin.Biller != nil {
+		handler := s.admin.Biller.WebhookHandlerFunc(ctx)
+		if handler != nil {
+			// TODO add observability middleware and rate limiter
```

Comment:
1. May be we don't add observability to this endpoint as Orb for example cannot selectively deliver events so it might be too much logging
2. Should we move webhook endpoints to worker server ?

Reply:
1. How many events are we talking about? In general, it would be nice to have logging for all requests as it also helps us understand load and where errors are occurring.
2. No the worker server is really only a fallback for health checks, it's not supposed to be exposed on the public internet. This is the service's external server, so this is the place to capture webhooks.

Reply:
Few events during org creation and then few events on subscription addition and changes. Then events on every invoice issue, creation, payment etc. Its just that since Orb cannot selectively deliver events, may be its ok. I can add observability.